### DISCUSSION TITLE
Prevent page from jumping

### DISCUSSION THREAD

**Discussion by iholzleitner** [jspsych/jsPsych#3462] [Q&A]
Hi there

I'm still on jsPsych v7 and experiencing an issue with the survey-text plugin. I have a long preamble, and on loading the page, the browser will automatically jump to the bottom of the page (to the text entry field). This means my participants do not read all of the instructions I need them to see.

Is there perhaps an easy fix (that doesn't require me to migrate to jsPsych v8)?

Thanks :)

**Comment by trig11** [comment]
I've had a similar issue, and this here worked to some degree:

```
on_load: function(){
			window.scrollTo(0,0); 
		},
```

However, this seemed to work fine in all browsers that I tested except for Firefox (although it's been a while since used, so maybe that has changed).
